site/source/tutorials/1030_routing/index.md

```diff
@@ -0,0 +1,183 @@
+
+Normally this is perfectly acceptable, however there are scenarios, where it is necessary to explicitly define a widget for an `index` match. To support this more advanced configuration, the first argument also accepts an object that can be used to specify these components explicitly.
+
+```ts
+const ExampleOutlet = Outlet({
+	index: MyIndexComponent,
+	component: MyComponent,
+	error: MyErrorComponent
+}, 'outlet-name');
+```
+
+We use this option for both the `BannerOutlet.ts` and `WorkerContainerOutlet.ts`, as we do not want the widgets to render for routes unless the are an exact match for the selected route.
+
+{% instruction 'Add the following code to `WorkerContainerOutlet.ts`.' %}
+
+{% include_codefile 'demo/finished/biz-e-corp/src/outlets/WorkerContainerOutlet.ts' %}
+
+{% instruction 'Add the follow code to `BannerOutlet.ts`.' %}
+
+{% include_codefile 'demo/finished/biz-e-corp/src/outlets/BannerOutlet.ts' %}
+
+Now, we can the swap out the widgets for the newly created `outlets`. Notice we haven't needed to changed any of wrapped widgets!
+
+{% instruction 'First, swap the widget imports for the outlet imports in `App.ts`.' %}
+
+{% include_codefile 'demo/finished/biz-e-corp/src/widgets/App.ts' lines:8-10 %}
+
+{% instruction 'Then, swap the widget usages for their outlets.' %}
+
+{% include_codefile 'demo/finished/biz-e-corp/src/widgets/App.ts' lines:50-55,59-61 %}
+
+Running the application now, should display the `Banner.ts` by default but also enable routing to the other widgets using the `/#directory` and `/#new-widget` routes.
+
+Next, we will add a side menu with links for the created outlets.
+
+{% section %}
+
+## Adding Links
+
+{% task 'Add a sidebar menu to the application.' %}
+
+In this section we will be using the `Link` component provided by `@dojo/routing`, to create link elements with a `href` for an outlet name. A `label` for the `Link` is can be passed as children and `param` values for the outlet can be passed to `Link` component using the `params` property.
+
+```ts
+w(Link, { to: 'outlet-name', params: { paramName: 'value' } });
+
+```
+
+{% instruction 'Add the `Link` import in `App.ts`.' %}
+
+{% include_codefile 'demo/finished/biz-e-corp/src/widgets/App.ts' line:4 %}
+
+{% instruction 'Replace the render function in `App.ts`.' %}
+
+{% include_codefile 'demo/finished/biz-e-corp/src/widgets/App.ts' lines:37-55,59-64 %}
+
+Now, the links in the side menu can be used to navigate around the application!
+
+{% section %}
+
+## Dynamic Outlet
+
+{% task 'Add filters to the `WorkerContainer.ts` widget and create outlet.' %}
+
+Finally, we are going to enhance the `WidgetContainer.ts` with a filter on the workers last name. To do this we need to use the `filter` outlet configured in the first section. The key difference for the `filter` outlet is that the path is using a placeholder that indicates a path parameter, `{filter}`.
```

Comment:
workers -> either worker's or workers' (depending if you mean on one work or many workers, as it is possessive)